### ISSUE TITLE
Discovery Day v3 | Explore Miro

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#2502]
**Challenge**
Explore [Miro](https://miro.com/), an "online whiteboard" to see how it might support various DTS activities:

~- Explore the **diagramming functionality** and templates — use the [DTS Service Delivery workflow](https://www.draw.io/#G1CAxAypAhB8dve1FnthN4LvIyVfKpTFKA) as an example~ #2583 
- Explore **ideation and brainstorming** functionality — consider remote meetings and design thinking exercises 

**Deliverables**
- Screenshots
- Notes 
- Recommendation

**Comment by amenity** [comment]
Unfortunately I didn't end up having a lot of time to devote to this and got carried away playing with Miro's Zapier integrations — in particular the [Github zap](https://miro.com/marketplace/github/) —  thinking that it could be a great tool to present a high-level DTS roadmap if we imported `Index` issues. 

It has been years since I played around with Zapier, so it was a treat to revisit it. Unfortunately Github's Zapier triggers are limited:
![Screen Shot 2020-05-06 at 10 00 33 PM](https://user-images.githubusercontent.com/1463708/81249781-1c1a7480-8fe5-11ea-9937-a88e6f29cc92.png)
As such, I wasn't able to figure out a way to selectively bring Github issues into Miro. I think it would have to be done via a CSV ... which is essentially the same thing we have set up using Roadmunk already. 

I also experimented using the Miro app to digitize post it notes and was impressed by the results:
![Screen Shot 2020-05-06 at 10 47 45 PM](https://user-images.githubusercontent.com/1463708/81252711-3dcb2a00-8fec-11ea-82c7-15e65ab7d2eb.png)

Overall I really liked Miro and the documentation. It sounds like the diagramming functionality is the most pertinent to current ATD needs in this era of fully-remote work, so I made this #2583 issue to continue looking into that.
